std-bits/table/src/main/java/org/enso/table/data/column/storage/StringStorage.java

```diff
@@ -51,12 +50,7 @@ public TextType getType() {
   @Override
   public Storage<?> fillMissing(
       Value arg, StorageType commonType, ProblemAggregator problemAggregator) {
-    if (arg.isString()) {
-      TextType newType = TextType.maxType(type, TextType.preciseTypeForValue(arg.asString()));
-      return fillMissingHelper(arg, new StringBuilder(size(), newType));
-    } else {
-      return super.fillMissing(arg, commonType, problemAggregator);
-    }
+    return super.fillMissing(arg, commonType, problemAggregator);
```

Comment:
This override seems to now just delegate to the super, shall we remove the override altogether?

Reply:
Yes good call. And fillMissingHelper is also un-required now too. Have changed it.